README.md

```diff
@@ -204,7 +204,10 @@ Make sure that you have a Vulkan ready driver and install the [LunarG Vulkan SDK
 
 #### Linux
 
-Make sure that you have a Vulkan ready driver and install the [LunarG Vulkan SDK](https://lunarg.com/vulkan-sdk/). You also have to add the library and layers to your path. Have a look at my [post](http://askubuntu.com/a/803110/77183) if you are unsure how to do that.
+Install a Vulkan driver for your graphics hardware of choice, and the [LunarG Vulkan SDK](https://lunarg.com/vulkan-sdk/) for the [ICD](https://github.com/KhronosGroup/Vulkan-Loader) and [Validation Layers](https://github.com/KhronosGroup/Vulkan-ValidationLayers).
+
+- Arch Linux: https://wiki.archlinux.org/title/Vulkan
+- Ubuntu: https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=vulkan-loader&searchon=sourcenames
```

Comment:
Any other distro that we should mention here? I could so far only find useful docs for Arch, the rest is just package manager searches...

Reply:
Meh, this seems fine. People can PR additional docs if they like. Maybe even omit the Ubuntu link; I bet the loader and most drivers are shipped by default there, and a novice user might not be sure what to do with this info.

Reply:
Maybe a comment along the lines of "For other distros, check your package manager"?